WeatherNotFound/src/main/java/me/learning/weathernotfound/initializer/WeatherNotFoundInitializer.kt

```diff
@@ -0,0 +1,16 @@
+package me.learning.weathernotfound.initializer
+
+import android.content.Context
+import androidx.startup.Initializer
+import me.learning.weathernotfound.presentation.WeatherNotFound
+
+
+class WeatherNotFoundInitializer : Initializer<WeatherNotFound> {
```

Comment:
Because this class is must reachable only inside of library, change the visibility of it to `internal`.

also put this class in `presentation` layer.